.travis.yml

```diff
@@ -19,13 +19,18 @@ matrix:
         - php: hhvm
 
 before_install:
+    - if [ -f "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" ]; then mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini-disabled"; fi;
```

Comment:
Right, we could do that... and then just create a new configuration file when we need to re-enable.